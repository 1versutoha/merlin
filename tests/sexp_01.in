["find", "use", "sexplib"]
["tell", "source", "module Test_sexp = struct "]
["tell", "source", "type t = int with sexp "]
["tell", "source", "end"]
["tell", "end"]
["errors"]
["dump", "env"]
