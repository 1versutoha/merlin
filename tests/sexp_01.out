["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"result":true}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",[]]
["return",[["","module Test_sexp :\n  sig\n    type t = int\n    val sexp_of_t : t -> Sexplib.Sexp.t\n    val t_of_sexp : Sexplib.Sexp.t -> t\n  end"]]]
